tests/integration_tests/corpus/test_update.py

```diff
@@ -177,7 +177,7 @@ def test_update_corpus_non_super_admin(client: TestClient, admin_user_header_tok
     )
     assert response.status_code == status.HTTP_403_FORBIDDEN
     data = response.json()
-    assert data["detail"] == "User admin@cpr.org is not authorised to UPDATE a CORPORA"
+    assert data["detail"] == "User admin@cpr.org is not authorised to UPDATE a CORPUS"
```

Comment:
Just double checking - we don't have anything that reads this and uses it for anything? Assuming it's just for logging.